scripts/start/readCheckScenarioConfig.R

```diff
@@ -179,11 +181,12 @@ readCheckScenarioConfig <- function(filename, remindPath = ".", testmode = FALSE
     # sort out known but forbidden names from unknown
     unknownColumnNames <- setdiff(unknownColumnNames, names(forbiddenColumnNames))
     if (length(unknownColumnNames) > 0) {
-      message("\nAutomated checks did not find counterparts in main.gms and default.cfg for these columns in ",
+      message("\nAutomated checks did not find counterparts ",
+              if (isFALSE(coupling)) "in main.gms and default.cfg for these columns in ",
```

Comment:
```suggestion
              if (isFALSE(coupling)) "did not find counterparts in main.gms and default.cfg for these columns in ",
              if (isTRUE(coupling)) "found these REMIND switches in ",              
```

Reply:
Ah, ok. But why do then throw the warning "You are not allowed to specify REMIND switches in coupled config."?

Reply:
Well. The subtle difference between telling the user that he used REMIND switches which he is not allowed to, and telling him that he used some switches that I don't know where they come from and tell him that he isn't allowed to use REMIND switches. Should I turn it into: "Maybe you specified REMIND switches, which you are not allowed to in coupled config."

Reply:
Still a bit confusing for me, but carry on, I don't want to get in the way any longer.